ruby/hangman/hangman.rb

```diff
@@ -0,0 +1,215 @@
+# game class
+
+class Hangman
+	attr_reader :solution_array, :game_result, :game_valid, :guess_array, :counter, :game_array
+	attr_accessor :guess, :repeat_guess
+
+	def initialize solution
+		@solution_array = solution.downcase.split('')
```

Comment:
Wayyyyyyy to many instance variables are being made on initialization.  Think about what each one does, why you need it, and how necessary is it really to save its state for every instance.  You should be passing methods and returning values to get a lot of the information.  There are too many layers of complexity here.

Consider initializing the game taking a solution word as a param and maybe a total guess amount and guess storage.  That should be about all you need to run this game